src/snackBar/Main.java

```diff
@@ -0,0 +1,48 @@
+package snackBar;
+
+public class Main
+{
+	public static void main(String[] args) 
+	{
+	// create Jane and Bob
+		Customer jane = new Customer("Jane", 45.25);
+		Customer bob = new Customer("Bob", 33.14);
+
+
+		// Create 3 vending machines
+		VendingMachine food = new VendingMachine("Food");
+		VendingMachine drink = new VendingMachine("Drink");
+		VendingMachine office = new VendingMachine("Office");
+
+		// Create 5 snacks
+		Snack chips = new Snack("Chips", 36, 1.75, food.getId());
+		Snack chocolate = new Snack("Chocolate Bar", 36, 1.00, food.getId());
+		Snack pretzel = new Snack("Pretzel", 30, 2.00, food.getId());
+		Snack soda = new Snack("Soda", 24, 2.50, drink.getId());
+		Snack water = new Snack("Water", 20, 2.75, drink.getId());
+
+		jane.buySnack(3, soda);
+		System.out.println("Janes cash on hand: " + jane.getCashOnHand());
```

Comment:
```suggestion
		System.out.println(jane.getName() + "cash on hand: " + jane.getCashOnHand());
```
just to be alittle more dynamic